homeassistant/components/lgthinq/trigger.py

```diff
@@ -0,0 +1,52 @@
+# SPDX-FileCopyrightText: Copyright 2024 LG Electronics Inc.
+# SPDX-License-Identifier: LicenseRef-LGE-Proprietary
```

Comment:
This file is a nice example; it is fully copied from an integration that Home Assistant already has in its codebase. You cannot claim copyright on this code.